cluster/kube-down.sh

```diff
@@ -28,6 +28,18 @@ echo "Bringing down cluster using provider: $KUBERNETES_PROVIDER"
 
 verify-prereqs
 teardown-monitoring
+
+# If logging with Fluentd to Elasticsearch was enabled then
+# delete the pods and service and remove the firewall rule
+# for accessing Elasticsearch and Kibana.
+if [[ "${ENABLE_NODE_LOGGING-}" == "true" ]]; then
```

Comment:
I prefer to think of the polymorphic nature of it - a platform might define a different implementation of setup-logging - that is the API.  fluent/elastic is the impl